sdk/internal/src/main/kotlin/uk/gov/onelogin/criorchestrator/sdk/internal/config/DefaultConfig.kt

```diff
@@ -30,6 +30,10 @@ private val defaultConfig =
                     key = IdCheckWrapperConfigKey.EnableManualLauncher,
                     Config.Value.BooleanValue(false),
                 ),
+                Config.Entry<Config.Value.BooleanValue>(
```

Comment:
Please could we add a test for this by adding an item to `DefaultConfigTest.missingEntry`?